yogstation/code/modules/mob/living/simple_animal/drone/interaction.dm

```diff
@@ -0,0 +1,71 @@
+/////////////////////
+//DRONE INTERACTION//
+/////////////////////
```

Comment:
You never include this file in the .dme?

Reply:
Fixed